README.md

```diff
@@ -1 +1,207 @@
-# ppa
\ No newline at end of file
+# ppa
+
+## Repo Overview
+
+This repo is uses GitHub Actions & Workflows to generate / upload Debian & Ubuntu builds to the https://launchpad.net/~ansible PPAs. See https://github.com/ansible-community/ppa/issues/1 for the latest PPA / Ansible / Ubuntu version matrix.
+
+## Repo Structure
+
+### Branches
+
+Along with the `main` branch, there is currently a branch for each major Ansible release. This is due to the build differences between versions of Ansible.
+
+```
+% git branch
+  ansible-2.8
```

Comment:
2.8's last release was in april 2021 and has since been EOL but I notice that we have releases as far back as 1.9 on the PPA.

This makes me wonder a few things like:
- What should we do with the 2.8 branch, if anything ?
- Should the EOL releases be removed/archived/disabled from the PPA ?

Reply:
That's a good question, I don't believe they cause any problems hanging around. Previous discussions leaned towards leaving them around just in case someone needed an old version for some reason. I could probably be convinced either way. 🤷